Tests/StarknetTests/Utils/DevnetClient.swift

```diff
@@ -0,0 +1,319 @@
+            "--account_dir",
+            accountDirectory,
+            "--wallet",
+            "starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount",
+            "--max_fee",
+            "0"]
+        
+        let result = runStarknetCli(
+            name: "Contract deployment",
+            command: "deploy",
+            args: params.joined(separator: " "))
+
+        let array = result.components(separatedBy: CharacterSet.newlines)
+        let tx = getTransactionResult(lines: array)
+
+        //TODO assertTxPassed
+        //assertTxPassed(tx.hash)
+        return tx
+    }
+    
+    public func declareContract(contractPath: String) -> TransactionResult {
+        let params = [
+            "--contract",
+            contractPath,
+            "--account_dir",
+            accountDirectory,
+            "--wallet",
+            "starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount"]
+        let result = runStarknetCli(
+            name: "Contract declare",
+            command: "declare",
+            args: params.joined(separator: " "))
+
+        let array = result.components(separatedBy: CharacterSet.newlines)
+        let tx = getTransactionResult(lines: array, offset: 2)
+
+        //TODO assertTxPassed
+        //assertTxPassed(tx.hash)
+        return tx
+    }
+
+    
+    private func runStarknetCli(name: String, command: String, args: String) -> String {
+        let process = Process()
+        
+        let outputPipe = Pipe()
+        let errorPipe = Pipe()
+        process.standardOutput = outputPipe
+        process.standardError = errorPipe
+        process.arguments = [
+            "-l",
+            "-c",
+            "/Users/jakub/.asdf/shims/starknet \(command) \(args) --gateway_url \(gatewayUrl) --feeder_gateway_url \(feederGatewayUrl) --network alpha-goerli"]
+        
+        process.launchPath = "/bin/sh"
+        process.standardInput = nil
+        process.launch()
+        process.waitUntilExit()
+        
+        let outputData = outputPipe.fileHandleForReading.readDataToEndOfFile()
+        let errorData = errorPipe.fileHandleForReading.readDataToEndOfFile()
+        
+        let output = String(decoding: outputData, as: UTF8.self)
+        let _ = String(decoding: errorData, as: UTF8.self)
+        
+        return output
+    }
+    typealias AccountDetailsResponse = [String: [String: AccountDetails]]
+    
+    public func readAccountDetails(accountName: String) -> AccountDetails {
+        let result = AccountDetails(privateKey:0, publicKey:0, address:0, salt:0)
+        let filename = "\(accountDirectory)/starknet_open_zeppelin_accounts.json"
+        
+        do {
+            let contents = try String(contentsOfFile: filename)
+            if let data = contents.data(using: .utf8) {
+                if let response = try? JSONDecoder().decode(AccountDetailsResponse.self, from: data) {
+                    return (response["alpha-goerli"]?[accountName])!
```

Comment:
Let's get rid of force unwrapping here. Either by making this method throwable, or by returning optional